content/guides/hanami/v2.3/database/relations.md

```diff
@@ -139,7 +141,7 @@ These relationships are called **associations** and they are defined within a `s
 
 ### One-to-Many
 
-booksidbigintpublisher_idbiginttitletextisbn13varchar(13)language_idbigintpagesintpublication_datedatepublishersidbigintnametext
+{{% one-to-many %}}
```

Comment:
What are these lines doing? In the deploy preview they're just showing exactly as written.

<img width="517" height="120" alt="Screenshot 2025-11-17 at 9 13 21 pm" src="https://github.com/user-attachments/assets/456f8da4-dd27-4b67-afa5-9703856c84b8" />

Reply:
Ah! These are actual diagrams [in our current guides](https://guides.hanamirb.org/v2.3/database/relations/#associations):

<img width="469" height="323" alt="Screenshot 2025-11-17 at 9 14 20 pm" src="https://github.com/user-attachments/assets/514b0e23-d4b7-4bf2-b1a2-4270b75c8f76" />

Did you have any thinking about how we handle these? It would be nice to preserve them, but if we had to temporarily lose them, that would be acceptable, I think. Let's make sure we add an issue to bring them back?